uttut/tests/test_expand_by_entities.py

```diff
@@ -16,11 +17,17 @@
 class ExpandByEntitiesTestCase(TestCase):
 
     def setUp(self):
+        self.gen_old_data()
+        self.gen_new_data()
+
+    def gen_old_data(self):
         self.utterance = '我想訂明天從紐約飛到新加坡的機票'
         self.entities = [
-            Entity(name='日期', value='明天', start=3, end=5, replacements=['下禮拜二']),
+            Entity(name='日期', value='明天', start=3,
+                   end=5, replacements=['下禮拜二']),
```

Comment:
Avoid coding style that is inconsistent with our repo

Reply:
e1ac824